com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/CsrfHeaders.java

```diff
@@ -1,9 +1,11 @@
 package com.sap.cloud.lm.sl.cf.web.security;
 
-interface CsrfHeaders {
+class CsrfHeaders {

```

Comment:
maybe should also be part of Constants like File Entry fields?

Reply:
There is not exist Constants class in maven module "web" and I think these constants should be kept in "web" module